src/docfx/lib/schema/EnumDependenciesSchema.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    internal class EnumDependenciesSchema : Dictionary<string, Dictionary<JToken, EnumDependenciesSchema>>
```

Comment:
this is a recursive data structure, did you intent it to support more than 2 levels of dependencies?

Reply:
Yes, because this rule in v2 also supports recursive model. i think we need to be consistent with it

Reply:
add a little description about the data structure, what the key and value means

Reply:
sure